src/main/java/utils/command/TerminateCommand.java

```diff
@@ -10,6 +10,7 @@ public class TerminateCommand extends Command {
     @Override
     public void execute(CardList cardList, TagList tagList, UserInterface ui, IDataStorage storage)
             throws InkaException {
+        storage.save(cardList);
```

Comment:
This makes sense, but what if the user exits the program abruptly (e.g., Ctrl-C or closing the window)

Should we just save this after every command that modifies the cards?